website/client/src/components/shops/quests/questDialogContent.vue

```diff
@@ -4,9 +4,15 @@
       class="quest-image"
       :class="'quest_' + item.key"
     ></div>
-    <h4 class="title">
+    <h3 class="text-center">
       {{ itemText }}
-    </h4>
+    </h3>
+    <div
+      v-if="group && leader"
+      class="leader-label"
+    >
+      <span>Quest-Owner: </span><user-label :user="leader" />
```

Comment:
Hardcoded English text. Also, double-check with TC/SO, I'm guessing this shouldn't be a hyphenated term